content/articles/implementing-flask-api-using-linked-list/index.md

```diff
@@ -0,0 +1,520 @@
+
+### Introduction
+Understanding data structures is an important aspect when developing applications. As real-world applications consider space and time complexities when designing applications to save on the memory and time taken to run specific programs.
+
+Despite the useful trade-offs, they offer to the pool of developers, most rarely use them. These make their programs and software not being to the quality and standards expected and can make organizations incur huge losses in terms of budgeting for storage and consumer loading time of these applications.
+
+In this article, we'll build a hands-on Flask API using a linked list data structure. In the process we'll see how applicable this data structure is and the benefits a user can acquire by using it.
+
+### Table of Contents
+1. [Prerequisites](#prerequisites)
+2. [Introduction to Flask](#introduction-to-flask)
+3. [Understanding Linked List data structure](#understanding-linked-list-data-structure)
+4. [Implementing Flask API using Linked List](#implementing-flask-api-using-linked-list)
+5. [Testing the API using Postman](#testing-the-api-using-postman)
+6. [Conclusion](#conclusion)
+
+
+### Prerequisites
+Before we start building the application, you should have the latest version of [Python](https://www.python.org/downloads/) installed or at least version `Python 3.6+`.
+
+To help install the necessary packages and dependencies you should have `pip` installed. If not check out the [link](https://pip.pypa.io/en/stable/installation/) to have it in your machine depending on your operating system.
+
+We'll be using the DB Browser for SQLite to visually check the database and its tables. You can download it from [here](https://sqlitebrowser.org/).
+
+For testing APIs you will need [Postman](https://www.postman.com/downloads/). Depending on your preference you can choose a text editor of your choice.
+
+There are a number such as [Visual Studio Code](https://code.visualstudio.com/download), [Atom](https://atom.io/), [Sublime](https://www.sublimetext.com/3), [Vim](https://www.vim.org/download.php) among others.
+
+For this tutorial, I'll be using Ubuntu 20.04 and Vim text editor.  
+
+### Introduction to Flask
+Flask is a micro-framework of Python that is used to create custom web applications. It is lightweight and does not come with any bundled packages and dependencies enabling developers to be on the wheel while building any application on it.
```

Comment:
Use the correct punctuation. The second sentence seems to be too long. Break it down using commas or make it to two sentences.